### PR TITLE
Postgres

### PR DESCRIPTION
**Pull request by ajayyy**

### CODE REVIEW

File: src/databases/Postgres.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import { Logger } from '../utils/logger';
+import { IDatabase, QueryType } from './IDatabase';
+import { Pool } from 'pg';
+
+import fs from "fs";
+
+export class Mysql implements IDatabase {
```
Comment (opl-) [inline src/databases/Postgres.ts:7]: Postgres is not MySQL. Class should be renamed.

File: src/databases/Sqlite.ts [hunk 2]
```diff
@@ -12,37 +12,35 @@ export class Sqlite implements IDatabase {
...
         const preparedQuery = this.db.prepare(query);
 
         switch (type) {
             case 'get': {
-                return preparedQuery.get(...params);
+                if (params) {
+                    return preparedQuery.get(...params);
+                } else {
+                    return preparedQuery.get();
+                }
```
Comment (opl-) [inline src/databases/Sqlite.ts:24]: These two operations should be equivalent. Why are they being split up?

Comment (ajayyy) [reply src/databases/Sqlite.ts:24]: "exploding` can't be done when the object is undefined. It gives an error.

Comment (opl-) [reply src/databases/Sqlite.ts:24]: Ah. In that case, you can either use a default argument value: `params?: any[] = {}`

Or use an operator: `params = params ?? []` or `params = params || []`

**Pull request merged**